bftengine/src/bcstatetransfer/DBDataStore.cpp

```diff
@@ -3,7 +3,6 @@
 #include "storage/db_interface.h"
 #include "Serializable.h"
 
-using bftEngine::bcst::BLOCK_DIGEST_SIZE;
```

Comment:
Can you remove/replace all occurrences of BLOCK_DIGEST_SIZE if DIGEST_SIZE is having same value?

Reply:
Changed.